en/orm/database-basics.rst

```diff
@@ -136,7 +136,7 @@ like::
             'username' => 'my_app',
             'password' => 'sekret',
             'database' => 'my_app',
-            'encoding' => 'utf8',
+            'encoding' => 'utf8mb4',
```

Comment:
I'd outcomment this line like in cakephp/app. Otherwise I'd write: 

    'encoding' => 'utf8mb4', // For non MySQL/MariaDB we recommend 'utf8'

Reply:
The above is for a MySQL connection though.